.github/workflows/build.yml

```
@@ -763,3 +763,9 @@ jobs:
           tag: ${{ github.ref }} # == github.event.release.tag_name
           file_glob: true
           file: ${{ matrix.artifact_name }}.7z.*
+
+      - name: Run release test workflow
+        uses: ./.github/workflows/release-test.yml
+        with:
+          version: ${{ github.ref }} # == github.event.release.tag_name
+          repo_url: ${{ format('{0}/{1}', github.server_url, github.repository) }}  # このリポジトリのURL
```

調べた限りでは、`workflow_call` で再利用できるようにした workflow の呼び出しは、action を使う時とは異なり steps 内に記述するのではなく、新しく job を作って使う、というように読めました。

参考： https://docs.github.com/ja/enterprise-cloud@latest/actions/using-workflows/reusing-workflows#calling-a-reusable-workflow

> You call a reusable workflow by using the uses keyword. Unlike when you are using actions within a workflow, you call reusable workflows directly within a job, and not from within job steps.
> 再利用可能なワークフローを呼び出すには、uses キーワードを使用します。ワークフロー内でアクションを使用しているときとは異なり、再利用可能なワークフローはジョブステップ内からではなく、ジョブ内で直接呼び出します。

利用例として以下のようなコードが書いてありました：

```yml
jobs:
  call-workflow-passing-data:
    uses: octo-org/example-repo/.github/workflows/reusable-workflow.yml@main
    with:
      username: mona
    secrets:
      envPAT: ${{ secrets.envPAT }}

```

従って、おそらく以下のようにすれば動く気がしています（ただ、自分も使ったことのない機能なので完全に自信があるわけではないです……）。

```suggestion
  run-release-test-workflow:
    uses: ./.github/workflows/release-test.yml
    with:
      version: ${{ github.ref }} # == github.event.release.tag_name
      repo_url: ${{ format('{0}/{1}', github.server_url, github.repository) }}  # このリポジトリのURL
```

---

おお、本当だ･･･ありがとうございます、ちょっと修正してみます！

---

@PickledChair needsとifを足してみました！！